pages/in/widget/iscn/index.vue

```diff
@@ -233,6 +240,7 @@ export default {
     if (!Object.keys(query).length) {
       return redirect('https://docs.like.co/developer/international-standard-content-number-iscn/web-widget');
     }
+    fingerprint = fingerprint.split(',');
```

Comment:
I think you can use a new variable `fingerprints`, because after `split(',')` event a string without `,` will turns into a single element array

Reply:
Should also guard case where `fingerprint` is `undefined`

Reply:
ok, proposed change:
1. use new variable fingerprint`s`
2. put the fingerprints definition after fingerprint to guard undefined case:
`if (!fingerprint) {
      return error({ statusCode: 400, message: 'INVALID_FINGERPRINT' });
    }
    const fingerprints = fingerprint.split(',');`